frontend/app/activeGame.js

```diff
@@ -1,11 +1,138 @@
+    const endTurnButton = document.getElementById('end-turn');
+    endTurnButton.onclick = () => { generateUserDetails.endTurn(sendJSON.sendJSON); };
+    endTurnButton.disabled = true;
+    // console.log(`id Test:${id}`);
+    // console.log(`turn Test:${turn}`);
+    if (turn === String(id)) {
+        generateUserDetails.enableGameInterface();
+    }
+}
+
+/**
+ * Called when a playerBalance event happens.
+ * Takes in all balance event data and checks it against the users id.
+ * If there is a match it updates their id.
+ *
+ * @param playerBalanceEvent The data received from the event
+ */
+export function onPlayerBalance(playerBalanceEvent) {
+    const data = JSON.parse(playerBalanceEvent.data);
+    let balance = '';
+    // console.log(`Balances:${data}`);
+    data.forEach((item) => {
+        // console.log(item);
+        if (String(item[0]) === String(id)) {
+            ({1: balance} = item);
+            document.getElementById('balance').innerHTML = balance;
+        }
+    });
+}
+
+/**
+ * Function for displaying the monopoly board onscreen.
+ * @param playerList The list of players in the game
+ */
+
+export function displayBoard(playerList) {
+    // console.log('displayBoard called');
+    document.getElementById('content').innerHTML = '<canvas id="gameBoard" height="800" width = "800" style="position: absolute; left: 0 ; top: 0 ;z-index : 0;"></canvas>';
+
+    // creates a canvas with player id and layer i.
+    // layer 0 = background image, last layer = game info
+    // creates a token for each player on their canvas.
+    for (let i = 1; i <= playerList.length; i += 1) {
+        createCanvas(playerList[i - 1], 'content', i);
+        control.movePlayer(playerList[i - 1], 0);
+    }
+    createCanvas('game-info', 'content', playerList + 1);
+    const c = document.getElementById('gameBoard');
     const ctx = c.getContext('2d');
     const img = new Image();
-    img.src = 'monopoly.jpeg';
-    ctx.drawImage(img, 0, 0);
+    img.onload = () => {
+        ctx.drawImage(img, 0, 0);
+    };
+    img.src = 'monopoly.jpg';
+}
+
+// takes in id for canvas, node to append to(<div id="content">), layerNumber
+function createCanvas(canvasID, appendToNode, layerNumber) {
+    const canvas = document.createElement('canvas');
+    canvas.id = canvasID;
+    canvas.height = 800;
+    canvas.width = 800;
+    canvas.style.position = 'absolute';
+    canvas.style.left = 0;
+    canvas.style.top = 0;
+    canvas.style.zIndex = layerNumber;
+    document.getElementById(appendToNode).append(canvas);
+}
+
+
+function enableActiveGameListeners() {
+    const eventSource = getEventSource();
+    eventSource.addEventListener('playerMove', onPlayerMove);
+    eventSource.addEventListener('playerTurn', onPlayerTurn);
+    eventSource.addEventListener('playerBalance', onPlayerBalance);
+    // eventSource.addEventListener('gameEnd', disableActiveGameListeners);
+}
+
+/*
+function disableActiveGameListeners(gameEndEvent) {
```

Comment:
Should this still be commented?

Reply:
It was commented in the branch I merged. I never uncommented it. Will do!

Reply:
So this is throwing errors because it's never used!

Reply:
Oh, the linter doesn’t like it? That’s frustrating. Leaving it commented for the moment sounds good – we can use it when we add the win condition.